src/ethereum/vm/instructions.py

```diff
@@ -44,21 +68,313 @@ def add(evm: Evm) -> None:
     """
     evm.gas_left = subtract_gas(evm.gas_left, GAS_VERY_LOW)
 
-    x = pop(evm.stack)
-    y = pop(evm.stack)
-
+    x, y = pop_multiple(evm.stack, 2)
```

Comment:
I'm not sure I prefer this over popping each individual element. With `pop`, it didn't feel like I even needed to look at the function definition. With `pop_multiple` I am slightly less certain about the implementation. @SamWilsn thoughts?

Reply:
I dislike `push_multiple` and `pop_multiple`.